web/content/release-nightly.adoc

```diff
@@ -29,8 +29,5 @@ Use the top menu bar for navigation for all guides.
 * link:/{FOREMAN_VER}/Release_Notes/index-foreman-el.html[Release notes for Foreman on RHEL/CentOS]
 * link:/{FOREMAN_VER}/Release_Notes/index-foreman-deb.html[Release notes for Foreman on Debian/Ubuntu]
 * link:/{FOREMAN_VER}/Release_Notes/index-katello.html[Release notes for Katello on RHEL/CentOS]
-
-// Upgrading guides are not ready for non-Katello
```

Comment:
The reason I previously had this visual separation was indicate grouping as well as a TODO. It also made sure that the order was already set: you only needed to uncomment them once they were done.

I wonder what others think. Does it make sense to keep that in?

Reply:
In `3.7` we don't have these. It should be identical to both places, no? IMO, guides are ready for `non-Katello`

Reply:
I now see https://github.com/theforeman/foreman-documentation/pull/2334 wasn't merged yet, but I consider that an oversight. After that the guides are properly hidden.